### ISSUE TITLE
Accessibility - checkbox

### ISSUE THREAD

**Issue by GingerC75** [siemens/ix#2397]
### Prerequisites

- [x] I have read the [Contributing Guidelines](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md).
- [x] I have not leaked any internal/restricted information like screenshots, videos, code snippets, links etc.

### What happened?

When an interactive elemnt is supposed to be a checkbox it cannot have the role chekcbox and button present. These kind of elements are not keyboard operable and thus violate W3C WAI WCAG2.2. success criteria 2.1.1 and 4.1.2. 

### What type of frontend framework are you seeing the problem on?

Angular (NgModule)

### On which version of the frontend framework are you experiencing the issue?

latest

### Which version of iX do you use?

v4

### Code to produce this issue.

```shell
<label>  Enable flex mode offen
2 von 2
Page DOM Strukturansicht
Ebene 33
<input aria-checked="false" type="checkbox"> 
gewählt
1 von 4
   <button class style="margin-right: 0px;">  Enable flex mode offen
```

**Comment by AndreasBerliner** [comment]
Hello @GingerC75 
I am closing this issue as it could not be reproduced with the checkbox component.
If you continue to face the problem please feel free to reopen it.
